### PR TITLE
51 to update status order after callback is completed

### LINKED ISSUE #51: to update status order after callback is completed

### PR DESCRIPTION
**Pull request by egch**

### CODE REVIEW

File: sushibar-backend-app/src/main/java/org/virtualsushibar/backend/app/dao/document/OrderStatus.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.virtualsushibar.backend.app.dao.document;
+
+public enum OrderStatus {
+    ORDER_CONFIRMED,
```
Comment (sarthak-97) [inline sushibar-backend-app/src/main/java/org/virtualsushibar/backend/app/dao/document/OrderStatus.java:4]: We can add a status as Pending order, for the retry attempt

File: sushibar-backend-avro/src/main/resources/avro/Order.avsc [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+    {
+      "name": "orderId",
+      "type": {
+        "type": "string",
+        "avro.java.string": "String"
+
+      }
+    },
+    {
+      "name": "amount",
```
Comment (sarthak-97) [inline sushibar-backend-avro/src/main/resources/avro/Order.avsc:16]: Shall we use double or float for the amount to have some decimal precision

**Review by sarthak-97 — APPROVED** [review]

File: sushibar-backend-app/src/main/java/org/virtualsushibar/backend/app/dao/document/OrderStatus.java (hunk 1 above)
Comment (egch) [reply sushibar-backend-app/src/main/java/org/virtualsushibar/backend/app/dao/document/OrderStatus.java:4]: there is already this: ORDER_NOT_CONFIRMED

File: sushibar-backend-avro/src/main/resources/avro/Order.avsc (hunk 2 above)
Comment (egch) [reply sushibar-backend-avro/src/main/resources/avro/Order.avsc:16]: it was conceived as a product identifier, such as a can of coke, a given pizza, therefore a non-decimal quantity. In the future we can think about extending it

**Pull request merged**